test/Microsoft.TemplateEngine.Cli.UnitTests/EndToEndTestBase.cs

```diff
@@ -22,7 +22,7 @@ public void Run(string args, params string[] scripts)
 #else
             string configuration = "Release";
 #endif
-            string harnessPath = Path.Combine(dir, "..", "..", "..", "..", "bin", "Microsoft.TemplateEngine.EndToEndTestHarness", configuration, "netcoreapp3.1");
+            string harnessPath = Path.Combine(dir, "..", "..", "..", "..", "bin", "Microsoft.TemplateEngine.EndToEndTestHarness", configuration, Path.GetFileName(dir));
```

Comment:
unsure if I understand this one 100%.  Can we either talk about it briefly or get someone else to approve this particular item?

Reply:
Made few changes to make it clearer what intention of that change is